### PR TITLE
Calc

### PR DESCRIPTION
**Pull request by vialikanau-aliaksei**

### CODE REVIEW

File: js/calc.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      result = a - b;
+      break;
+    case '*':
+      result = a * b;
+      break;
+    case '/':
+      result = a / b;
+      break;
+    case 'int/':
+      result = Math.round(a / b);
```
Comment (NikitaSedyx) [inline js/calc.js]: `Math.round(6.9 / 1) == 7` but it should be 6

**Review by NikitaSedyx — CHANGES_REQUESTED** [review]

File: js/calc.js (hunk 1 above)
Comment (vialikanau-aliaksei) [reply js/calc.js]: why 6?  Change rounding to (a/b)^0 ?

File: index.html [hunk 2]
```diff
@@ -1,7 +1,24 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+  <meta charset="UTF-8">
+  <title>Calc</title>
 </head>
 <body>
+<form name="main">
```
Comment (NikitaSedyx) [inline index.html]: please move each task to new folder, thank

Comment (NikitaSedyx) [reply index.html]: also please add spaces for nested tags

File: js/calc.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+      result = a - b;
+      break;
+    case '*':
+      result = a * b;
+      break;
+    case '/':
+      result = a / b;
+      break;
+    case 'int/':
+      result = (a / b) ^ 0;
```
Comment (NikitaSedyx) [inline js/calc.js]: 1/0 ^ 0 = 0
but I want to be Infinity
please look at Math function

**Review by NikitaSedyx — CHANGES_REQUESTED** [review]

**Review by NikitaSedyx — APPROVED** [review]

**Pull request merged**